translations/bnpchs-2018-final-bermain-dengan-bentuk.html

```diff
@@ -0,0 +1,91 @@
+<h3>Description</h3>
+<p style="text-align: justify;">
+  Jelix is indeed naughty! This time he did something mischievous.
+  Jelix did not tidy up the legos he had played and left them on the floor.
+  Seeing this, Jelix's parents scolded Jelix and ordered him to tidy up immediately.
+</p>
+
+<p style="text-align: justify;">
+  We can represent those legos in a two-dimensional grid where each cell contains a lego.
+  Formally if there is a lego located in $(x, y)$, then the legos adjacent to it are in $(x+1, y)$, 
```

Comment:
```suggestion
  Mathematically if there is a lego located in $(x, y)$, then the legos adjacent to it are in $(x+1, y)$, 
```
just for preserving the originality of the statement